app/services/universal/promises.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+/**
+ * bluebird.props with native promises
+ * @param {Object} props
+ * @returns {Promise}
+ */
+module.exports.props = (props) => {
```

Comment:
You can even go one step further with this function, like this: 

```
module.exports.props = (props) => {
  return Promise.all(Object.keys(props).map((key) => Promise.resolve(props[key]).then((res) => ({ [key]: res }))))
    .then((resolvedArray) => resolvedArray.reduce((memo, oneRes) => {
      const key = Object.keys(oneRes)[0];

      memo[key] = oneRes[key];

      return memo;
    }, {}));
};
```

Reply:
cool thanks